docs/guides/getting-started/setup/_fragments/_tauri-init.mdx

```diff
@@ -10,28 +10,28 @@ Our Tauri CLI uses Cargo under the hood so you rarely need to interact with it d
 If you haven't installed the Tauri CLI yet you can do so with with one of the below commands. Aren't sure which to use? Check out the [FAQ entry].
 
 <Tabs groupId="package-manager">
-<TabItem value="npm" label="npm" default>
+<TabItem value="npm" label="NPM" default>
 
 ```shell
 npm install --save-dev @tauri-apps/cli
 ```
 
 </TabItem>
-<TabItem value="yarn" label="yarn">
+<TabItem value="yarn" label="Yarn">
 
 ```shell
 yarn add -D @tauri-apps/cli
 ```
 
 </TabItem>
-<TabItem value="pnpm" label="pnpm">
+<TabItem value="pnpm" label="PNPM">
```

Comment:
same thing for `pnpm`

Reply:
Pssssshhhhhh what are capitalisation rules? Node ecosystem just throw them out the window

Reply:
yeeeeeet